lib/endpoint/tags/table.js

```diff
@@ -130,6 +130,14 @@ function rowToParameter (i, el) {
     params.type = 'integer[]'
   }
 
+  if (/^array$/.test(params.type)) {
+    params.type = 'string[]'
+  }
+
+  if (/array of.*strings/.test(params.type)) {
+    params.type = 'string[]'
+  }
```

Comment:
this is not what the issue was about, see the open PR with the failing test at https://github.com/octokit/routes/pull/77/files. It’s about to changing the implementation to make the test pass when you run 

```
TEST_URL=https://developer.github.com/v3/orgs/members/#create-organization-invitation ./node_modules/.bin/tap test/integration/single-endpoint-test.js
```

and

```
TEST_URL=https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization ./node_modules/.bin/tap test/integration/single-endpoint-test.js
```

Reply:
Oh wow! So the issue is, that `array` and ( `string[]` or `number[]` ) are exactly the same thing but the test still fails. Right?

Reply:
no they are not the same thing. In the case of the `team_ids` parameter, the values of the array are integers. In the case of the scopes, they are strings. I guess we can default to `string[]` and make an exception for `parameter.name === 'team_ids'`

Reply:
Roger that. ;)
And i meant that they both are array, so in that sense. :p
On it.